pyproject.toml

```diff
@@ -52,6 +56,23 @@ exclude = '''
   )/
 )
 '''
+
+[tool.mypy]
+python_version = "3.8"
+warn_return_any = true
+warn_unused_configs = true
+
+[[tool.mypy.overrides]]
+module = "pyarr"
+disallow_untyped_defs = true
+
+[tool.isort]
```

Comment:
I would recommend using isort's built-in profile for `black`

```
[isort]
profile=black
```

Reply:
ah yes didnt realise i had missed that!